spec/instant.html

```diff
@@ -402,12 +400,12 @@ <h1>
       </h1>
       <dl class="header">
         <dt>description</dt>
-        <dd>It returns its argument _item_ if it is already a Temporal.Instant instance, converts _item_ to a new Temporal.Instant instance if possible, and throws otherwise.</dd>
+        <dd>Converts _item_ to a new Temporal.Instant instance if possible, and throws otherwise.</dd>
       </dl>
       <emu-alg>
         1. If _item_ is an Object, then
           1. If _item_ has an [[InitializedTemporalInstant]] internal slot, then
```

Comment:
We can now collapse this into "If _item_ has an [[InitializedTemporalInstant]] or [[InitializedTemporalZonedDateTime]] internal slot".